scene/gui/popup_menu.cpp

```diff
@@ -1019,7 +1019,7 @@ void PopupMenu::activate_item(int p_item) {
 	//hide all parent PopupMenue's
 	Node *next = get_parent();
 	PopupMenu *pop = Object::cast_to<PopupMenu>(next);
-	while (pop) {
+	while (pop && p_item >= 0 && p_item < items.size()) {
```

Comment:
That's odd, there's already an `ERR_FAIL_INDEX` at the beginning, so it should never reach this if `p_item` is not within the `items.size()` boundary.